### PR TITLE
Add metrics for active operations on service layer.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review
@vekterli : FYI

### CODE REVIEW

**Comment by geirst** [comment]
I think @vekterli needs to review this as well as he is most familiar with the existing code.

File: storage/src/vespa/storage/persistence/filestorage/active_operations_metrics.cpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#include "active_operations_metrics.h"
+
+namespace storage {
+
+ActiveOperationsMetrics::ActiveOperationsMetrics(metrics::MetricSet* parent)
+    : MetricSet("active_operations", {}, "metrics for active operations at service layer", parent),
+      size("size", {}, "Number of concurrent active operations", this),
+      latency("latency", {}, "Latency for active operations", this)
```
Comment (geirst) [inline storage/src/vespa/storage/persistence/filestorage/active_operations_metrics.cpp:10]:
```suggestion
      latency("latency", {}, "Latency (in ms) for active operations", this)
```

**Review by geirst — APPROVED** [review]
👍

**Review by vekterli — APPROVED** [review]
:+1:

File: storage/src/vespa/storage/persistence/filestorage/active_operations_metrics.cpp (hunk 1 above)
Comment (toregge) [reply storage/src/vespa/storage/persistence/filestorage/active_operations_metrics.cpp]: Fixed.

**Review by vekterli — APPROVED** [review]

**Pull request merged**